index.html

```diff
@@ -92,6 +92,8 @@ <h1 id="version-info-id">v1.0.34</h1>
           friends! <br />Supported by Curious Learning.
         </p>
       </div> -->
+      <!-- Replace the canvas with a div for feedback text -->
+      <div id="feedback-text" class="feedback-text" style="z-index: 0;"></div>
```

Comment:
why is there a default style here of z-index? Why not move this to css file?

Reply:
removed this code this was on my debugging earlier. thanks for noticing it @bernhardccodev